app/src/main/java/org/mozilla/fenix/tabtray/TabTrayView.kt

```diff
@@ -113,6 +119,19 @@ class TabTrayView(
                     if (!hasLoaded) {
                         hasLoaded = true
                         tray.layoutManager?.scrollToPosition(selectedBrowserTabIndex)
+                        if (view.context.settings().accessibilityServicesEnabled) {
+                            GlobalScope.launch {
+                                delay(1000)
```

Comment:
This is obviously not ideal but I'm at a loss.  This works with over 100 tabs on my Pixel 4, but I don't have an older device to test, and arbitrary delays aren't ideal.

Reply:
We should probably use the view lifecycle scope to make sure this gets properly cancelled if you quickly back out